lib/plugins/aws/provider.js

```diff
@@ -577,7 +586,7 @@ class AwsProvider {
                   { $ref: '#/definitions/awsCfInstruction' },
                   { $ref: '#/definitions/awsCfIf' },
                   { $ref: '#/definitions/awsCfSelect' },
-                  { $ref: '#/definitions/awsCfToJsonString' },
+                  { $ref: '#/definitions/awsCfBase64' },
```

Comment:
This feels as breaking change, as if I understand correctly, we will no longer support `'Fn::ToJsonString'` here

Reply:
Fn::ToJsonString is now covered by `awsCfInstruction` (indirectly, because it includes `awsCfFunction`)